core/src/main/resources/settings.yml

```diff
@@ -68,6 +68,9 @@ CustomBlocks:
   block_correction: NMS # Valid types are NMS and LEGACY
   use_legacy_noteblocks: false # Should only be enabled when updating from Oraxen 1.x
 
+TuneBlock:
+  reimplement-note-block-features: true # Makes regular note blocks work with custom blocks, Also makes custom block instrument feature work
```

Comment:
settings are also not defined in settings.yml anymore, just havent removed this file